Update "Add boards to Arduino IDE" / "Uninstall boards from Arduino IDE" [HC-1606] (#439)
arduino/help-center-content: content/Software Support/Installation/Uninstall-boards-from-Arduino-IDE.md

> The following instructions to uninstall is not limited to the core in this example; it equally would be applicable to both official Arduino and third party cores.

Reviewer: ```suggestion
The following instructions to uninstall are not limited to the core in this example; they are equally applicable to both official Arduino and third-party cores.
```

In this article:

- [Uninstall using the Boards Manager](#uninstall_board)
- [Uninstall by directly deleting files](#delete_files)

---

## Uninstall using the Boards Manager{#uninstall_board}

The Boards Manager provides the most convenient way to remove unwanted or unused boards.

> 1. In the menu bar, select *Tools > Board > Boards Manager...* or you can also click on the ![Boards Manager icon](img/symbol_board.png) button in the sidebar.

Reviewer: ```suggestion
1. In the menu bar, select *Tools > Board > Boards Manager...*, or click on the ![Boards Manager icon](img/symbol_board.png) button in the sidebar.
```
